test/src/main/java/org/apache/accumulo/test/AuditMessageIT.java

```diff
@@ -428,6 +428,7 @@ public void testDeniedAudits() throws AccumuloSecurityException, AccumuloExcepti
     try {
       Scanner scanner = auditConnector.createScanner(OLD_TEST_TABLE_NAME, auths);
       scanner.iterator().next().getKey();
+      scanner.close();
```

Comment:
I think that in many cases like this, the following try-with-resources syntax is preferred:

```java
try (Scanner scanner = auditConnector.createScanner(OLD_TEST_TABLE_NAME, auths)) {
  scanner.iterator().next().getKey();
}
```

This is a simpler syntax that doesn't require an explicit call to the AutoCloseable's close method.

Reply:
Oh you are so right, I didn't even think of just putting the scanner objects in the try loops if there is one.  some of the scanners weren't in try-with-resources already so I just closed them where they were. I will make that change where possible.